rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -1339,4 +1352,21 @@ private void add_patch_releases(MockHttpServletRequestBuilder requestBuilder) th
                 .content(this.objectMapper.writeValueAsString(releaseIds))
                 .header("Authorization", "Bearer " + accessToken)).andExpect(status().isCreated());
     }
+
+    private void add_patch_releases_with_project_release_relation(MockHttpServletRequestBuilder requestBuilder)
+            throws Exception {
+        ProjectReleaseRelationship projectReleaseRelationship1 = new ProjectReleaseRelationship(
+                ReleaseRelationship.REFERRED, MAINLINE);
+        ProjectReleaseRelationship projectReleaseRelationship2 = projectReleaseRelationship1.deepCopy()
```

Comment:
Is cloning necessary here? May be it can be written as `new ProjectReleaseRelationship(ReleaseRelationship.STANDALONE, MainlineState.SPECIFIC)
                .setComment("Test Comment 2")` ?

Reply:
Modified code